omero/sysadmins/unix/install-web.rst

```diff
@@ -68,7 +68,7 @@ display a maintenance page.
       $ omero web clearsessions
 
   It is recommended to call this command on a regular basis, for example 
-  as a :download:`daily cron job <walkthrough/omero-web-cron>`, see
```

Comment:
This feels like something is being lost.

Reply:
Could the download have been moved to _this_ page?

Reply:
The example did not add much. I can look at adding them to the doc. 
They should be in omeroweb-install in theory but that will require reviewing the generation of the doc.